src/Workspaces/Core/Portable/Experiments/IExperimentationService.cs

```diff
@@ -36,5 +36,6 @@ internal static class WellKnownExperimentNames
         public const string LspTextSyncEnabled = "Roslyn.LspTextSyncEnabled";
         public const string RemoveUnusedReferences = "Roslyn.RemoveUnusedReferences";
         public const string LSPCompletion = "Roslyn.LSP.Completion";
+        public const string UnnamedSymbolCompletionDisabled = "Roslyn.UnnamedSymbolCompletionDisabled";
```

Comment:
It's weird that this is a negative option shouldn't it be `Roslyn.UnnamedSymbolCompletion` whose default is `true`?

Reply:
it's a feature flag.  we can't make a default for it.  that would force us to have the experiment always running for all time, in order for this to work.  

So this is the negative so we can say:  by default, this runs.  but an experiment can be *enabled* to turn it off. As opposed to:  this is off, and we must always run an experiment forever to turn it on.

Reply:
We could add handling for this in `VisualStudioExperimentationService` for this but that does seem like overkill I suppose. We own all implementations of `IExperimentationService` so it's up to us to decide what an option being enabled means.